applications/nrf_desktop/app.overlay

```diff
@@ -0,0 +1,12 @@
+/ {
```

Comment:
OK, how would you like this to be done instead?

Reply:
Please note the CMakeLists.txt changes I've just pushed on top

Reply:
The app is suppose to run on multiple boards. I cannot say if this configuration will be correct for all of them. Even if we don't use CC310 it may be that some client will. Each board configuration has its overlay in the configuration directory. It can be added there.
`applications/nrf_desktop/configuration/BOARD/dts.overlay`

Reply:
Any individual board overlay will override this default, because DTC_OVERLAY_FILE is an ordered list, and later files will override earlier ones.

Reply:
yes, but it fall out how the configuration is designed. If you think it's to much work to copy it to these few places I can do it.

Reply:
I have pushed the commit with a change `a28e9aa0ba2e376373e4d35d0ff6e9e6f7b8f12a`. Also cleaned up a conf files of 833 and 820. I guess you should squash it with the other commit or I can to it if you like.

Reply:
Great, thanks. Now squashed and your sign-off added.